controlplane/eks/controllers/awsmanagedcontrolplane_controller.go

```diff
@@ -232,7 +232,7 @@ func (r *AWSManagedControlPlaneReconciler) reconcileNormal(ctx context.Context,
 		// Wait for the cluster infrastructure to be ready before creating machines
 		if !managedScope.Cluster.Status.InfrastructureReady {
 			managedScope.Info("Cluster infrastructure is not ready yet")
-			return ctrl.Result{}, nil
+			return ctrl.Result{RequeueAfter: 5 * time.Second}, nil
```

Comment:
I think it's reasonable to requeue and not wait for the resync period.

Shall we make this duration configurable via a flag with 5 mins as the default?

Reply:
Hey @richardcase, thanks for the comment, do you mean pass the flag as container argument?

Reply:
I've added the flag, please review it and give me feedback.

Also, I think is better to set 1 minute as default, in all my tests, Infrastructure become ready between 30 seconds and one minute